### DISCUSSION TITLE
8.4.0 (Profile action)

### DISCUSSION THREAD

**Discussion by T-Troll** [T-Troll/alienfx-tools#367] [Announcements]
###  Changes into this release:

`AlienFX-GUI`:
- New: It's now possible to run app/script/etc. at profile change:
  - Switch to "Profiles" tab and set the action name by type it or choose it using "Set" button
  - Check "run action"
  - Action will be triggered at the every profile switch
  - You can use application, script, document or shell command as an action.
  - This option is incompatible (and will be disabled) if "Only then active" selected - running action change active window, so it comes to endless loop in this case.
- Power mode check after resume from sleep added - profile will be re-selected in case power mode changed during sleep
- Tooltip generation changed to prevent possible exception.
- Profile UI optimized both visual (for better understanding) and logical (for performance).
- Bug fixed: Some profile settings (f.e. device effect) do not apply at app start/resume from sleep.

<hr /><em>This discussion was created from the release <a href='https://github.com/T-Troll/alienfx-tools/releases/tag/'>8.4.0 (Profile action)</a>.</em>

**Comment by aron0614** [comment]
My cpu gpu power mode and battery power mode will change during playing games how I can avoid this issue

**Reply by JazzyJazzjazzy** [reply]
Change the power mode for your Windows PC

Choose the power mode that works for you and what you want to do on your Windows 11 PC. This lets you determine what’s important to you—getting the best battery life, best performance, or a balance between the two.
To change the power mode, select Start > Settings > System > Power & battery. For Power mode, choose the one you want.

You may want to try HIGH PERFORMANCE mode.

Check that BATTERY SAVER mode is OFF or at a very low value.

Step 1: Press Win + I to open the Settings app.

Step 2: Go to System > Power & battery.

Step 3: Click the Turn on now button to turn it on. However, if battery saver is already turned on and you want to turn it off, there will be a Turn off now button. You can click it to disable battery saver on Windows 11.

**Reply by T-Troll** [reply]
... Or just install something like BatteryMode and do it (and much more) in one click ^_^
